typings/pinejs-client-core.d.ts

```diff
@@ -114,30 +117,53 @@ type BaseExpandFor<T> = { [k in keyof T]?: object } | StringKeyof<T>;
 
 export type Expand<T> = BaseExpandFor<T> | Array<BaseExpandFor<T>>;
 
-export interface PineOptions {
-	$select?: string[] | string | '*';
+export type ODataMethod = 'GET' | 'POST' | 'PUT' | 'PATCH' | 'DELETE';
+
+interface ODataOptionsBase {
+	$orderby?: OrderBy;
+	$top?: string;
+	$skip?: string;
+	$select?: string | string[] | '*';
+}
+
+export interface PineOptions extends ODataOptionsBase {
 	$filter?: object;
 	$expand?: object | string;
-	$orderby?: OrderBy;
-	$top?: number;
-	$skip?: number;
 }
 
-export interface PineOptionsFor<T> extends PineOptions {
+export interface PineOptionsFor<T> extends ODataOptionsBase {
 	$select?: Array<StringKeyof<T>> | StringKeyof<T> | '*';
 	$filter?: Filter<T>;
-	$expand?: Expand<T>;
+	$expand?: Expand<T> | string;
```

Comment:
Should this be part of the `export type Expand<T>....` defined above?

Reply:
Oh actually it seems that this is a leftover from the rebase, since we now have `type BaseExpandFor<T> = { [k in keyof T]?: object } | StringKeyof<T>;`.
Nice heads-up @sradevski

Reply:
Updated